rss-client/src/app/User/Admin/admin-page/admin-page.component.html

```diff
@@ -8,10 +8,10 @@
         </ng-template>
         <ng-template [ngSwitchCase]="'inventory'">
             <admin-inventory (navigateEvent)="navigateTo($event)"></admin-inventory>
-        </ng-template>
-        <!--default case when there are no matches -->
-        <ng-template [ngSwitchDefault]>
-            <!-- <div class="container">
+    </ng-template>
+    <!--default case when there are no matches -->
+    <ng-template [ngSwitchDefault]>
+    <!-- <div class="container">
```

Comment:
Still not sure why the indent changes; also where are these divs closed?

Reply:
there is not a good reason for the indent changes, i don't know why did those got moved
there are closing divs on lines 25 and 26 I believe